tic_tac_toe/game.py

```diff
@@ -63,42 +78,106 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
+    
+    set_of_combos = ( #row wins
```

Comment:
@Psy-Kosh I totally agree with your assessment here. The goal was to come back and refactor to something a bit less imperative and a bit more expressive.